packages/audio-native/scripts/clean.mjs

```
@@ -0,0 +1,12 @@
+import { execSync } from "child_process";
+import { platform } from "os";
+
+const isWindows = platform() === "win32";
+
+if (isWindows) {
+  console.log("Cleaning native addon (node-gyp clean)...");
```

ESLint

[error] 'console' is not defined.

(no-undef)